.dockerignore

```diff
@@ -0,0 +1,11 @@
+.env
+.venv
+.git
+.vscode
+ansible-inventory.ini
+ansible-playbook.yml
+test_app.py
+test_view_mdodel.py
+Dockerfile
+
+
```

Comment:
You've got good coverage here - one extra thing we might want is any cache folders, like `**/__pycache__`

Also worth noting that the Dockerignore file is subtly different to .gitignore (because of course it is!) so this isn't actually ignoring some of these files - e.g. `test_app.py` still shows up in your container. The explanation:
* In a `.gitignore`, this would be doing exactly what you want
* But in `.dockerignore`, `file_name` _only_ ignores a file with that name at the _root_ of the build context
* So instead we need to specify `**/test_app.py`

For that reason, I tend to default to using `**/<whatever>` liberally in Dockerfiles over `<whatever>` because it's typically what we want